juicy-select.html

```diff
@@ -38,19 +38,24 @@
                     if (this.multiple) {
                         this.$select.setAttribute("multiple", "multiple");
                     }
+                    this.$select.addEventListener('change', this.selectChange);
                 }
-                this.set("multiple", this.$select.hasAttribute("multiple"));
+                else {
+                    this.set("multiple", this.$select.hasAttribute("multiple"));
+                }
+                if (this.options.length && this.$select) {
+                    this.generateOptions();
+                }
+                this.addEventListener("change", this.selectChange);
```

Comment:
Do you need both event listeners, one on line 41 and one on line 49?

Reply:
Good catch.